README.md

```diff
@@ -150,20 +150,50 @@ We collaborate closely with the design and product team. The design team provide
 
 ## Deployment
 
+### Git Branching Model
+
+- When a Github issue is started, create a branch with the `feature/` prefix. Every push to branches with this prefix will automatically deploy to the review AWS environment. See the [Review branches](#review-branches) section for details on how to access this build.
+
+- After a pull request for a `feature/` branch is approved, it is merged into the `staging` branch. This will trigger an automatic deployment to the staging AWS environment. Note that changes may take several minutes to take effect. The staging app is accessible at https://staging.fightpandemics.work.
+
+- When we are ready to release to production, a pull request is opened to merge the `staging` branch to `production`. After this pull request is approved and merged, it will trigger an automatic deployment to the production AWS environment. Note that changes may take several minutes to take effect. The production app is accessible at https://fightpandemics.com
+
+- A `hotfix/` branch is created if there is a critical fix that needs to be deployed to production as soon as possible, but staging has untested code that is not safe to deploy to production.
+    - This branch must be created off the `production` branch.
+    - Every push to `hotfix/` branches will automatically deploy to the review AWS environment. See the [Review branches](#review-branches) section for details on how to access this build.
+    - Two pull requests must be opened for hotfixes: one PR to merge into `production` and one PR to backmerge `production` into `staging` after the initial PR is merged to `production`.
+    - Hotfixes must be tested in the staging AWS environment before being merged to the `production` and `staging` branches. In order to deploy only the hotfix to the staging enviroment, create and push a git tag with a `hotfix-` prefix. This will trigger a deployment to the staging environment: `git tag -a hotfix-<GITHUB_ISSUE_NUMBER> && git push origin hotfix-<GITHUB_ISSUE_NUMBER>`.
+
+- All pull request merges to the `staging` branch, and all `hotfix/` merges to the `production` branch are done using the `Squash and Merge` strategy. This allows for a cleaner commit history by combining all commits in a branch into a single commit. The message for this squashed commit must use the following convention:
```

Comment:
If i've read the other comments correctly this part should also be changed and for `hotfix` we can't squash to prevent different commit hashes ...

Reply:
Yeah, that's why I changed [this](https://github.com/FightPandemics/FightPandemics/pull/1269/commits/80220c554a90eee5f7aa288722dc2be3680c7864#diff-04c6e90faac2675aa89e2176d2eec7d8R164) to instead backmerge from the `production` branch after the hotfix PR is merged to `production`, rather than merge directly from the hotfix branch into `staging`.